tools/dep_updaters/update-simdutf.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+set -e
+# Shell script to update simdutf in the source tree to a specific version
+
+BASE_DIR=$(cd "$(dirname "$0")/../.." && pwd)
+DEPS_DIR="$BASE_DIR/deps"
+SIMDUTF_VERSION=$1
+
+if [ "$#" -le 0 ]; then
+  echo "Error: please provide an simdutf version to update to"
+  echo "	e.g. $0 2.0.3"
+  exit 1
+fi
+
+echo "Making temporary workspace..."
+
+WORKSPACE=$(mktemp -d 2> /dev/null || mktemp -d -t 'tmp')
+
+cleanup () {
+  EXIT_CODE=$?
+  [ -d "$WORKSPACE" ] && rm -rf "$WORKSPACE"
+  exit $EXIT_CODE
+}
+
+trap cleanup INT TERM EXIT
+
+SIMDUTF_REF="v$SIMDUTF_VERSION"
+SIMDUTF_ZIP="simdutf-$SIMDUTF_VERSION.zip"
+
+cd "$WORKSPACE"
+
+echo "Fetching simdutf source archive..."
+curl -sL -o "$SIMDUTF_ZIP" "https://github.com/simdutf/simdutf/releases/download/$SIMDUTF_REF/singleheader.zip"
+tar -xf "$SIMDUTF_ZIP"
```

Comment:
```suggestion
unzip "$SIMDUTF_ZIP"
```

(it’s not a tar file)

Reply:
My bad! Thanks for the keen eyes :)